docs/DeveloperGuide.md

```diff
@@ -775,65 +807,72 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+#### Use case (UC12): Clear all appointments
 
-      Use case resumes at step 2.
+**MSS**
+
+1. User requests to clear all appointments.
+2. DocTrack clears all appointments.
+
+    Use case ends.
 
 <br>
 
+
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `17` or above installed.
-2.  Should work on any _reasonable system_ with good performance: common operation such as retrieving patient data must complete within 1 second, and complex operations must complete within 3 seconds.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-4.  Should not require installation
-5.  Must not operate with dependency on any remote server
-6.  No usage of DBMS
-7.  Main product file must not exceed 100MB
-8.  Documentation must not exceed 15MB
-9.  Product should be designed for typing-preferred consumers, offering a CLI experience
-10. Product should be designed for a single user.
-11. Product must function correctly on _standard resolutions_ and support scaling of 100%, 125%, 150%.
-12. Data must be persistent, with all changes saved immediately to local storage
-13. Data files must be in a format that can be edited manually by advanced users
-14. Data file must remain usable and intact even with invalid input from the application
-15. Errors must trigger clear, user-friendly messages
+1. Should work on any _mainstream OS_ as long as it has Java `17` or above installed.
+2. Should work on any _reasonable system_ with good performance: common operation such as retrieving patient data must complete within 1 second, and complex operations must complete within 3 seconds.
+3. Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
+4. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+5. Should not require installation
+6. Must not operate with dependency on any remote server
+7. No usage of DBMS
+8. Main product file must not exceed 100MB
+9. Documentation must not exceed 15MB
+10. Product should be designed for typing-preferred consumers, offering a CLI experience
+11. Product should be designed for a single user.
+12. Product must function correctly on _standard resolutions_ and support scaling of 100%, 125%, 150%.
+13. Data must be persistent, with all changes saved immediately to local storage
+14. Data files must be in a format that can be edited manually by advanced users
+15. Data file must remain usable and intact even with invalid input from the application
+16. Errors must trigger clear, user-friendly messages
```

Comment:
Should we add any specific non-functional requirements regarding performance? These are the notes made previously: 
Response Time:
Common operations (e.g., retrieving patient data) must complete within 1 second.
Complex operations (e.g., filtering records) must complete within 3 seconds.